packages/transition-backend/src/services/gtfsImport/ServiceImporter.ts

```diff
@@ -16,7 +16,7 @@ import ServiceCollection from 'transition-common/lib/services/service/ServiceCol
 import { _isBlank } from 'chaire-lib-common/lib/utils/LodashExtensions';
 import { ServiceImportData, GtfsImportData } from 'transition-common/lib/services/gtfs/GtfsImportTypes';
 import serviceLocator from 'chaire-lib-common/lib/utils/ServiceLocator';
-import { getUniqueServiceName } from 'transition-common/lib/services/service/ServiceUtils';
+import { getUniqueServiceName } from '../transitObjects/transitServices/ServiceUtils';
```

Comment:
Do we delete the old getUniqueServiceName ?

Reply:
> Do we delete the old getUniqueServiceName ?

Not yet, it is still used by lines/agency duplication while it is still in the frontend. Would be more trouble than it's worth to remove this now, knowing that all duplications will move to the backend shortly anyway.

Reply:
It could be interesting to do add a TODO there to not forget to delete it.